core/chains/evm/txmgr/config.go

```diff
@@ -25,12 +25,14 @@ type Config interface {
 	EvmGasPriceDefault() *assets.Wei
 	EvmGasTipCapMinimum() *assets.Wei
 	EvmMaxGasPriceWei() *assets.Wei
-	EvmMinGasPriceWei() *assets.Wei
-	EvmNonceAutoSync() bool
-	EvmRPCDefaultBatchSize() uint32
 	KeySpecificMaxGasPriceWei(addr common.Address) *assets.Wei
 }
 
+type EVMConfig interface {
```

Comment:
nit: Since this package is already evm, the name EVMConfig doesn't add much clarity on what this config contains.
Perhaps ChainConfig?

Also, look at overall config naming strategy, so it makes sense to reader. 
Example, the breakdown of config names in common/... should be consistent with name breakdown here. This similar config in common/... is named as ConfirmerChainConfig, and here, it is EVMConfig. 


I don't want to dictate naming here, and this is just an optional suggestion. Perhaps for now, do the breakdown first, and then in a later PR, consolidate everything into clearly defined sub-config names, consistent everywhere.

Reply:
Hey Prashant! No, I agree ChainConfig makes more sense here -- I've changed so it's consistent :)